common/src/main/java/net/bettercombat/logic/TargetHelper.java

```diff
@@ -58,7 +58,7 @@ public static Relation getRelation(PlayerEntity attacker, Entity target) {
     }
 
     public static boolean isAttackableMount(Entity entity) {
-        if (entity instanceof HostileEntity || isEntityHostileVehicle(entity.getEntityName())) {
+        if (entity instanceof HostileEntity || isEntityHostileVehicle(entity.getName().getString())) {
```

Comment:
Something to double-check - this might have to be getUuidAsString() instead of getName().getString().
getEntityName() returned the Entity's uuidString.
getName().getString() here returns "getClass().getName() + "@" + Integer.toHexString(hashCode())"
getUuidAsString() returns the Entity's uuidString.

It looks like Alex's Mobs hasn't updated to 1.20.4 yet so the connection here is still yet to be determined.

Reply:
> Something to double-check - this might have to be getUuidAsString() instead of getName().getString().
> getEntityName() returned the Entity's uuidString.
> getName().getString() here returns "getClass().getName() + "@" + Integer.toHexString(hashCode())"
> getUuidAsString() returns the Entity's uuidString.
> 
> It looks like Alex's Mobs hasn't updated to 1.20.4 yet so the connection here is still yet to be determined.

I'm not familiar with Minecraft apis, it's my first time

I just made changes so I can use the mod without any errors

I updated everything in gradle properties and build.gradle

I will wait once the owner replies and changes as required or uses his version if he rejects the PR

By the way, I'm not sure when isAttackableMount gets triggered